From 91db806946104f03d483371f3069adcff6ba24ca Mon Sep 17 00:00:00 2001
From: jschyra <jschyra@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 10 Jan 2014 10:57:56 +0000
Subject: [PATCH 0434/1000] MANTIS 30342  - You can now run MIPSKIC without
 AIDA:    - To run the script without AIDA, set the "-na" switch.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@633 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Tools/MIPSKIC/src/mipskic.bat | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Tools/MIPSKIC/src/mipskic.bat b/SelfServiceCommon/Tools/MIPSKIC/src/mipskic.bat
index 6c4151a9..0e14d2fe 100644
--- a/SelfServiceCommon/Tools/MIPSKIC/src/mipskic.bat
+++ b/SelfServiceCommon/Tools/MIPSKIC/src/mipskic.bat
@@ -26,6 +26,7 @@ set _passwd=
 set _output=%~dp0output
 set _configuration=
 set _mode=TEXT
+set _noAida=FALSE
 
 for %%a in (%*) do call :setopt %%a
 
@@ -56,7 +57,9 @@ echo Parse input file...
 FOR /F %%i in (%_hostsFile%) do (
 	echo.
     echo Processing %%i...
-    call aida.bat -H %%i -m %_mode% -o %_output%\
+	if "%_noAida%" == "FALSE" (
+		call aida.bat -H %%i -m %_mode% -o %_output%\
+	)
 	
 	if not "%_configuration%" == "" (
 			FOR %%a in (%_configuration%) do (
@@ -78,6 +81,7 @@ if /i [%1]==[-p] (set _setopt=_passwd& goto end)
 if /i [%1]==[-o] (set _setopt=_output& goto end)
 if /i [%1]==[-c] (set _setopt=_configuration& goto end)
 if /i [%1]==[-m] (set _setopt=_mode& goto end)
+if /i [%1]==[-na] (set _noAida=TRUE& goto end)
 
 set ERRORCHECK=%~1
 if /i [%ERRORCHECK:~0,1%]==[-] (
-- 
2.41.0.windows.1

